pkg/skaffold/schema/latest/v1/config.go

```diff
@@ -877,6 +877,12 @@ type Artifact struct {
 
 	// LifecycleHooks describes a set of lifecycle hooks that are executed before and after each build of the target artifact.
 	LifecycleHooks BuildHooks `yaml:"hooks,omitempty"`
+
+	// Line is the line number for the source skaffold.yaml file which has this artifact configuration info
+	Line int `yaml:"-"`
+
+	// Column is the column number for the source skaffold.yaml file for the artifact congfiguration info
+	Column int `yaml:"-"`
 }
```

Comment:
I wonder if it's worth moving these into a struct together. If we're adding lint rules for the artifact schema, I'm assuming that we'll probably be adding lint rules for other schema as well in the future. In that case, we could just drop that struct into whatever type we're adding lint rules for. Thoughts?